Protocol.md

```diff
@@ -100,7 +100,7 @@ To enable the reuse of already-existing packfiles and indexes, a custom format f
 is supported. The `prefetch` endpoint will return one or more packfiles of **non-blob** objects.  
 
 If the optional `lastPackTimestamp` query parameter is supplied, only packs created by the server
-after the specific Unix epoch time (approximately, �10 minutes or so) will be returned. Generally, these packs 
+after the specific Unix epoch time (approximately, ±10 minutes or so) will be returned. Generally, these packs 

```

Comment:
The +/- is correct, because this is a best effort to prefetch commits and trees. If we tried to make it a hard guarantee, then all the cache servers deployed behind a load balancer would have to synchronize with each other and make sure they're splicing the packfiles exactly the same way, and that is an unnecessary level of complexity.